app/services/attribute_asserter.rb

```diff
@@ -68,14 +68,18 @@ def ial2_service_provider?
   end
 
   def resolved_authn_context_result
-    @resolved_authn_context_result ||= begin
+    authn_context_resolver.resolve
```

Comment:
same suggestion as noted above about memoizing this value

Reply:
Ditto